tests/test_view_iter_objects.py

```diff
@@ -0,0 +1,99 @@
+from lona.pytest import eventually
+from lona import LonaView
+
+
+def setup_app(app):
+
+    @app.route('/view-1/')
+    class View1(LonaView):
+        pass
+
+    @app.route('/view-2/')
+    class View2(LonaView):
+        pass
+
+
+async def test_view_iter_objects(lona_app_context):
+    """
+    This test tests LonaView.iter_objects() with theese steps:
+
+     - Creating two views
+     - Opening 3 tabs on view 1
+     - Opening 2 tabs on view 2
+     - Count view objects of view 1 and 2 and check view classes
+     - Closing 1 tab on view 1
+     - Closing 1 tab on view 2
+     - Count view objects of view 1 and 2 and check view classes
+    """
+
+    from playwright.async_api import async_playwright
+
+    context = await lona_app_context(setup_app)
+
+    async with async_playwright() as playwright:
+        browser = await playwright.chromium.launch()
+        browser_context = await browser.new_context()
+
+        # view 1
+        view1_page1 = await browser_context.new_page()
+        view1_page2 = await browser_context.new_page()
+        view1_page3 = await browser_context.new_page()
+
+        await view1_page1.goto(context.make_url('/view-1/'))
+        await view1_page2.goto(context.make_url('/view-1/'))
+        await view1_page3.goto(context.make_url('/view-1/'))
+
+        # view 2
+        view2_page1 = await browser_context.new_page()
+        view2_page2 = await browser_context.new_page()
+
+        await view2_page1.goto(context.make_url('/view-2/'))
+        await view2_page2.goto(context.make_url('/view-2/'))
+
+        # count view objects
+        for attempt in eventually():
+            async with attempt:
+
+                # view 1
+                view_class = context.server.get_view_class(url='/view-1/')
```

Comment:
nit: Maybe it's better to use view class directly and avoid using `get_view_class`? But for this it's necessary to refactor `setup_app`.

Reply:
I don't propose to change an API. I propose to change the test to something like that
```py
assert len(list(View1.iter_objects())) == 3
for view_object in View1.iter_objects():
    assert view_object.__class__ == View1
```

Reply:
How would i get ``View1`` into the test function?